pravega-k8-log-collection/pravega-k8s-log.sh

```diff
@@ -0,0 +1,28 @@
+#/usr/bin/env bash
+set -euo pipefail
+mkdir -p /tmp/logdir
+output_dir="/tmp/logdir"
+kubectl get po,svc,pvc --all-namespaces > $output_dir/kubectl-get-po-svc-pvc-all-namespace_output
+IFS=$'\n'
+for i in $(cat $output_dir/kubectl-get-po-svc-pvc-all-namespace_output | grep -i pravega)
+do
+echo $i
+if [[ $i == *"po/"* ]] || [[ $i == *"pod/"* ]]; then
+    if [ `echo $i | awk '{print $5}'` != 0 ]; then     
+        kubectl -n `echo $i | awk '{print $1}'` logs `echo $i | awk '{print $2}'` -p > $output_dir/log_`echo $i | awk 'gsub(/\//,"_") {print $2}'`_previous.log
+    fi
+    kubectl -n `echo $i | awk '{print $1}'` logs `echo $i | awk '{print $2}'` > $output_dir/log_`echo $i | awk 'gsub(/\//,"_") {print $2}'`_recent.log
+    has_logs=$(kubectl -n `echo $i | awk '{print $1}'` exec -it `echo $i |  awk '-F'[/] '{print $2}'|awk '{print $1}'` -- ls -l /opt/pravega/logs 2>/dev/null | wc -l || echo "")
```

Comment:
Since you are comparing the number of lines in the output, I think I'd be good to use `echo 0` instead of `echo ""`.

Also, instead of `has_logs`, it'll be more descriptive if you use `log_count` or similar.

Reply:
Included necessary changes as discussed.